### PR TITLE
add japanese readme

### PR DESCRIPTION
**Pull request by U-top-Yoshida**
add japanese readme file
add a hyperlink to japanese readme on readme file

### CODE REVIEW

**Comment by keefertaylor** [comment]
+ Ryan who can help get this merged!

Thank you!

File: README-ja.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
+[![CircleCI](https://img.shields.io/circleci/build/github/xpring-eng/Xpring-JS?style=flat-square)](https://circleci.com/gh/xpring-eng/xpring-js/tree/master)
+[![CodeCov](https://img.shields.io/codecov/c/github/xpring-eng/xpring-js?style=flat-square)]((https://codecov.io/gh/xpring-eng/xpring-js))
+[![Dependabot Status](https://img.shields.io/static/v1?label=Dependabot&message=enabled&color=success&style=flat-square&logo=dependabot)](https://dependabot.com)
+
+# Xpring-JS
+
+Xpring-JSは、Xpring SDKのJavaScriptクライアント側ライブラリです。
+
+## 機能
+Xpring-JSは次の機能を提供します。
```
Comment (gbarr01) [inline README-ja.md]: Hi, @U-top-Yoshida , maybe use a colon here? 

> Xpring-JSは次の機能を提供します:

**Comment by gbarr01** [comment]
@U-top-Yoshida , just curious, do the docs ever explain (with katakana) how to pronounce "Xpring"?

**Comment by mDuo13** [comment]
The translation seems right to my (limited) knowledge.

File: README-ja.md (hunk 1 above)
Comment (U-top-Yoshida) [reply README-ja.md]: @gbarr01 Thank you for your review. yes, a colon will be there.

**Comment by U-top-Yoshida** [comment]
> @U-top-Yoshida , just curious, do the docs ever explain (with katakana) how to pronounce "Xpring"?

We read it, "スプリング" the same pronunciation "spring".

File: README-ja.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+Utils.isValidXAddress(rippleClassicAddress); // returns false
+Utils.isValidXAddress(rippleXAddress); // returns true
+Utils.isValidXAddress(bitcoinAddress); // returns false
+
+Utils.isValidClassicAddress(rippleClassicAddress); // returns true
+Utils.isValidClassicAddress(rippleXAddress); // returns false
+Utils.isValidClassicAddress(bitcoinAddress); // returns false
+```
+
+### X-Address エンコーディング
```
[the remark this answers is not in the document]
Comment (U-top-Yoshida) [reply README-ja.md]: @mDuo13 Thank you for your review, I need to edit it.

**Comment by U-top-Yoshida** [comment]
@gbarr01 @mDuo13 Hi, I have edited. Please check them. Thank you.

**Comment by gbarr01** [comment]
> > @U-top-Yoshida , just curious, do the docs ever explain (with katakana) how to pronounce "Xpring"?
> 
> We read it, "スプリング" the same pronunciation "spring".

I understand. But do the readers know this? Even English speakers are not always sure.

**Comment by U-top-Yoshida** [comment]
> > > @U-top-Yoshida , just curious, do the docs ever explain (with katakana) how to pronounce "Xpring"?
> > 
> > 
> > We read it, "スプリング" the same pronunciation "spring".
> 
> I understand. But do the readers know this? Even English speakers are not always sure.

Usually Japanese articles about Xspring write both english and katakana, like this 
Xspring（スプリング）
So I believe the readers know how to pronounce already. but it will be better to write both.

**Comment by gbarr01** [comment]
> * Ryan who can help get this merged!
> 
> Thank you!

@U-top-Yoshida , do you know what is needed to merge this PR?

**Comment by ryangyoung** [comment]
Hi @gbarr01 @U-top-Yoshida have all of the edits been made? I can help merge this if so.

**Comment by ryangyoung** [comment]
@keefertaylor can you help merge this?

**Comment by keefertaylor** [comment]
Hi! Sorry this took so long. 

Thank you for your contribution @U-top-Yoshida!

**Pull request merged**